cargo/cargo_build_script.bzl

```diff
@@ -121,6 +117,13 @@ def _build_script_impl(ctx):
         if cc_toolchain.sysroot:
             env["SYSROOT"] = cc_toolchain.sysroot
 
+    # In form build scripts of rustc flags
+    # https://github.com/rust-lang/cargo/issues/9600
+    env["CARGO_ENCODED_RUSTFLAGS"] = " ".join([
```

Comment:
Should this also include from `rustc_flags`?

Reply:
great idea! Done 😄